docs/release-instruction.md

```diff
@@ -0,0 +1,32 @@
+# Release instructions
+
+## Pre-release checks
+
+These should run successfully:
+* unit-tests,
+* `go mog tidy` (should not change any files);
+* `./prepare.sh /path/to/neofs-api/on/your/machine` (should not change any files).
+
+## Writing changelog
+
+Add an entry to the `CHANGELOG.md` following the style established there. Add an
+optional codename(for not patch releases), version and release date in the heading.
+Write a paragraph describing the most significant changes done in this release. Add
+`Fixed`, `Added`, `Removed` and `Updated` sections with fixed bug, new features and
+other changes.
+
+Open Pull Request (must receive at least one approval) and merge this changes.
+
+## Make a Github release
+
+Using Github's web interface create a new release with the same changes from changelog
+and publish it.
+
+## Close github milestone
+
+Close corresponding vX.Y.Z github milestone.
+
+## Post-release
+
+Prepare pull-request for updating `neofs-api-go` library in
```

Comment:
I suggest to not add it: lower level lib should not trigger the updating of the dependent lib.

Reply:
Yeah, let's drop `Post-release` section in library repository.

Reply:
dropped